tensorflow/tools/pip_package/setup.py

```diff
@@ -123,7 +123,7 @@ def standard_or_nightly(standard, nightly):
     # current release version. These also usually have "alpha" or "dev" in their
     # version name.
     # These are all updated during the TF release process.
-    standard_or_nightly('tensorboard >= 2.11, < 2.12',
+    standard_or_nightly('tensorboard >= 2.12, < 2.13',
```

Comment:
```suggestion
    standard_or_nightly('tensorboard >= 2.12-rc0, < 2.13',
```

Check name to be sure you pick the right now